### PR TITLE
Updated User Flags

### PR DESCRIPTION
**Pull request by Vonjoll**
There was a few user flags missing from the list, added these.
Unsure if this was intentional?

### CODE REVIEW

File: src/util/Constants.ts [hunk 1]
```diff
@@ -208,7 +208,11 @@ export enum USER_FLAGS {
   TEAM_USER = 1 << 10,
   // UNKNOWN 1 << 11 discord/discord-api-docs#1823
   SYSTEM = 1 << 12,
-  UNREAD_URGENT_MESSAGES = 1 << 13
+  UNREAD_URGENT_MESSAGES = 1 << 13,
+  BUG_HUNTER_LEVEL_2 = 1 << 14,
+  // ?? 1 << 15
```
Comment (bsian03) [inline src/util/Constants.ts:213]:
```suggestion
  // UNKNOWN 1 << 15
```

**Review by bsian03 — CHANGES_REQUESTED** [review]
This was partly intentional - Was trying to fill in gaps from Discord's documentation and stop at their documented highest values. I am planning to go through them again sometime in the future, since there a few other ones which I missed out as well. This isn't my top priority at the moment.

**Comment by Vonjoll** [comment]
No bother, just noticed them at a glance. 

Can go through them and add missing ones to relieve it from yourself. 

You mention about the highest discord documents - they document all the way to 17, other than 11 and 15?

**Comment by bsian03** [comment]
> Can go through them and add missing ones to relieve it from yourself.

Nah you don't need to - there's going to be a significant structure reorganisation which will be laid out in a roadmap which will be opened in an issue some time tomorrow, and I'd rather just wait until that was complete to go through everything in detail.

> You mention about the highest discord documents - they document all the way to 17, other than 11 and 15?

oops

File: src/util/Constants.ts (hunk 1 above)
Comment (Khaaz) [reply src/util/Constants.ts:213]: underage_deleted

Comment (Khaaz) [reply src/util/Constants.ts:213]: undocumented discord side, no need for us to document that as well

**Comment by Khaaz** [comment]
we also should have the comma dangle rule

**Comment by Vonjoll** [comment]
> we also should have the comma dangle eslint rule

What is a preference to this being set too? Never, always?

**Comment by Khaaz** [comment]
always, but it should be a separate PR

**Comment by bsian03** [comment]
I'll commit eslint later on today

**Review by bsian03 — APPROVED** [review]

**Review by Khaaz — APPROVED** [review]

**Pull request merged**